### PR TITLE
feat: update examples

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/examples/example01-editor.html [hunk 1]
```diff
@@ -22,7 +23,7 @@
     <!-- Editor -->
     <script src="../dist/toastui-editor-all.js"></script>
     <script type="text/babel" class="code-js">
-      const editor = new toastui.Editor({
+      const instance = new toastui.Editor({
```
Comment (dongwoo-kim) [inline apps/editor/examples/example01-editor.html]: `instance`라는 변수명은 썩 좋은 네이밍이 아닌데.. 바꾸는 이유가 뭔가요?

File: apps/editor/examples/example06-editor-with-chart-plugin.html [hunk 2]
```diff
@@ -37,7 +39,7 @@ <h2>Viewer</h2>
       const editor = new Editor({
         el: document.querySelector('#editor'),
         previewStyle: 'vertical',
-        height: '500px',
+        height: '5000px',
```
Comment (dongwoo-kim) [inline apps/editor/examples/example06-editor-with-chart-plugin.html]: 이건.. 오타인가요? 5000 이나 필요할 것 같지 않은데..

File: apps/editor/examples/example06-editor-with-chart-plugin.html [hunk 3]
```diff
@@ -47,7 +49,17 @@ <h2>Viewer</h2>
...
-        plugins: [chart]
+        plugins: [
+          [
+            chart,
+            {
+              minWidth: 100,
+              maxWidth: 600,
+              minHeight: 100,
+              maxHeight: 300
+            }
```
Comment (dongwoo-kim) [inline apps/editor/examples/example06-editor-with-chart-plugin.html]: 플러그인 적용 코드가 좀 못생겼는데.. ㅎ 
`chartOptions` 변수로 분리하는 건 어떨까요?

File: apps/editor/examples/example06-editor-with-chart-plugin.html (hunk 2 above)
Comment (seonim-ryu) [reply apps/editor/examples/example06-editor-with-chart-plugin.html]: .... 큰일날뻔 했네요..

File: apps/editor/examples/example01-editor.html (hunk 1 above)
Comment (seonim-ryu) [reply apps/editor/examples/example01-editor.html]: 다른 문서들도 인스턴스가 하나인 것들은.. `instance`로 표기를 맞추려고 했습니다만..

**Pull request merged**